CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [16.0.0]
+### Changed
+- **BREAKING**: Remove `syncTokens` method from the TokenListController ([#590](https://github.com/MetaMask/controllers/pull/590))
+  - This is breaking for any consumers of this method.
+- Enable default caching for token-service ([#594](https://github.com/MetaMask/controllers/pull/594))
+- Set tokenList to empty object in test networks ([#588](https://github.com/MetaMask/controllers/pull/588))
```

Comment:
This seems to be describing something internal - it doesn't make clear what the actual change was for consumers. It's not clear to me either from reading the PR.

My best guess is that this is a bug fix, and that the change was to clear the dynamic token list when switching to a network not supported by our token list API. Does that sound correct? cc @NiranjanaBinoy @danjm

Reply:
@Gudahtt that is correct